src/pages/Main/Main.js

```diff
@@ -1,9 +1,73 @@
-import React from 'react';
+import React, { Component } from 'react';
+import Item from '../Shopping/Item/Item';
+import './Main.scss';
+
+export default class Main extends Component {
+  constructor(props) {
+    super(props);
+    this.slideContainer = React.createRef();
+  }
+
+  state = {
+    slideIndex: 0,
+  };
```

Comment:
- ref 도 state 처럼 컨스트럭터를 거치지 않고 바로 선언하실 수 있습니다!
```suggestion
  state = {
    slideIndex: 0,
  };
  slideContainer = React.createRef();
```

Reply:
PR 템플릿 체크 주의하겠습니다 죄송합니다ㅜㅜ

Reply:
Ref 선언관련 수정했습니다.